forms-flow-web/src/components/AccessDenied/accessDenied.scss

```diff
@@ -0,0 +1,14 @@
+.access-denied-text {
+    font-family: Arial, Helvetica, sans-serif;
+    font-size: 32px;
+    letter-spacing: 0px;
+    color: #000000;
+    opacity: 1;
+  }
+  .access-denied{
+    font-family: Arial, Helvetica, sans-serif;
+    font-size: 16px;
+    letter-spacing: 0px;
+    color: #000000;

```

Comment:
please use variables if its available instead of color code

Reply:
resolved